2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_REPLICA_NODES",
+                    "value": "3"
```

Comment:
I think you need to rename this env var.  Right now it sounds like by setting it, I am controlling the number of replicas I will get, but you're not, you're setting the number of replicas to expect/wait for.